week03_model_free/display_server_preparation.sh

```diff
@@ -0,0 +1,47 @@
+echo "apt-get updating..."
```

Comment:
The script is missing a [shebang](https://en.wikipedia.org/wiki/Shebang_(Unix)) line, which in this case should probably be

```bash
#!/usr/bin/env bash
```

Without it, it will not work in non-`bash` shells (such as `fish`).

Also, I would suggest adding `set -e` or maybe even `set -ex`. This will cause the script to fail if any of the commands return nonzero exit code. Please refer to `man set` for more information (the `-e` option is close to the top).

Reply:
Yes, I read about "set -e", but some topics about this solution drawbacks were also read